juspay/hyperswitch
# feat(routing): Use Moka cache for routing with cache invalidation

## Type of Change



## Description

Enabled Moka cache for routing with cache invalidation


### Additional Changes





## Motivation and Context



## How did you test it?

Look for the pub-sub cache invalidation log
![image](https://github.com/juspay/hyperswitch/assets/76486416/e739ac71-2b2b-4103-9804-b4478c5d0e87)
![image](https://github.com/juspay/hyperswitch/assets/76486416/068743c0-a9d0-49af-859d-f23ab32eb2ec)

1. Update an existing Merchant connector account and look for Cgrpah cache invalidation log
```
curl --location --request POST 'http://localhost:8080/account/sarthak/connectors/mca_VaZpO5COiFDUIiMq7DFV' \
--header 'Content-Type: application/json' \
--header 'Accept: application/json' \
--header 'api-key: test_admin' \
--data-raw '{
    "connector_type": "fiz_operations",
    "metadata": {
        "city": "NY",
        "unit": "248"
    }
}'
```
2. Update default Routing config and check for routing cache invalidation log
```
curl --location --request POST 'http://localhost:8080/routing/default' \
--header 'api-key: dev_wnIRnsZGiq0NHNq3WskWv4Kw7Eg7InEHASiXrAXA4zM2MoJMvL2fCMDWNt8Ieg5I' \
--header 'Content-Type: application/json' \
--data-raw '["stripe"]'
```



## Checklist
